trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    super[0.9]
+  end
```

Comment:
- [ ] The correct name method here doesn't return the expected value; It only gives us the first character of the name as you can see in the attached image below 👇 
I am sure you meant to do `super[0..9]`?
![image](https://github.com/Cosmosha/school_library/assets/46839548/d340441e-a0e9-4082-b1f6-8caf544434ff)